homeassistant/components/samsungtv/strings.json

```diff
@@ -6,20 +6,19 @@
         "description": "Enter your Samsung TV information. If you never connected Home Assistant before you should see a popup on your TV asking for authorization.",
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
-          "name": "[%key:common::config_flow::data::name%]"
+          "name": "Name"
```

Comment:
Any reason why the standard string isn't being used?

Reply:
Done